controllers/spotify.controller.js

```diff
@@ -0,0 +1,48 @@
+
+const spotifyApi = require('../models/spotify');
+
+module.exports.home = (req, res) => {
+    res.render('home')
+}
+
+module.exports.searchArtist = (req, res) => {
+    
+    spotifyApi
+        .searchArtists(req.query.name)
+        .then(data => {
+            //console.log('The received data from the API: ', data.body);
+            res.render('artists/artist-search-results', { data: data.body })
+            //console.log("este es el dataBody:", data.body.artists)
+        })
+        .catch(err => console.log('The error while searching artists occurred: ', err));
+
+}    
+
+module.exports.getAlbum = (req, res) => {
+
+  spotifyApi.getArtistAlbums(req.params.artistId)
```

Comment:
mismo caso aqui podriamos hacer un 

```suggestion
const artistId = req.params.artistId
  spotifyApi.getArtistAlbums(artistId)
```